src/OauthTokenFileStorage.php

```diff
@@ -144,8 +142,12 @@ public function removeToken(): void {
   public static function oauthTokenPath() {
     static $token_path;
```

Comment:
Do not use static here, set the location in a property in the constructor of this class.

Reply:
Even if you do not DI the config factory, the location should be stored in a property.